elastalert/util.py

```diff
@@ -349,6 +352,44 @@ def replace_dots_in_field_names(document):
     return document
 
 
+def is_not_empty (var):
```

Comment:
If this was inverted to `is_empty` it would be easier to read. It would avoid the `if not is_not_empty()` which is a mouthful.

Reply:
Yeah I'll propose to have the two functions, it will be more understandable depending on the context (I use to have both, and it's often the case in utils libraries like Apache commons etc)

Reply:
Done